commands/dataTables/status.js

```diff
@@ -0,0 +1,5 @@
+const { dataTables } = require('../../lib/constants');
+
+module.exports = (program) => {
+  return require('../utils/status')(program, 'dataTables', dataTables);
```

Comment:
this file should not be in here, as we said there should be no status command.

Reply:
forgot about that lil' guy's existence